docs/horizen_eon/05-staking/03-how_to_stake_on_eon.md

```diff
@@ -3,9 +3,6 @@ title: How to Stake on EON
 ---
 
 # How to Stake on EON
-A forger is required to run a node. See [How to Locally Build and Deploy](../04-develop_and_deploy_smart_contracts/04-local_build_and_deploy.md) and [Forger Node Requirements](./04-forger_node.md) for information on creating and running a node.  
+You can stake ZEN on EON Forger nodes to participate in network security. Please stake to to a known forger node using the instructions noted in the section labeled Stake ZEN to Your Forger in the Forger Node tutorial. Note that this will instead be someone else's forger node instead of your own.
```

Comment:
```suggestion
You can stake ZEN on EON forger nodes to participate in network security. Please stake to a known forger node using the instructions noted in the section labeled Stake ZEN to Your Forger in the Forger Node tutorial. Note that this will instead be someone else's forger node instead of your own.
```

Reply:
Can `Stake ZEN to Your Forger in the Forger Node tutorial` be linked